README.md

```diff
@@ -0,0 +1,43 @@
+[![Build Status](https://travis-ci.org/andela/selene-ah-frontend.svg?branch=develop)](https://travis-ci.org/andela/selene-ah-frontend)
+[![Maintainability](https://api.codeclimate.com/v1/badges/798539e413b8f66d7c18/maintainability)](https://codeclimate.com/github/andela/selene-ah-frontend/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/798539e413b8f66d7c18/test_coverage)](https://codeclimate.com/github/andela/selene-ah-frontend/test_coverage)
+
+# Authors Haven - A Social platform for the creative at heart.
+
+## Vision
+
+Create a community of like minded authors to foster inspiration and innovation by leveraging the modern web.
+
+## Getting Started
+
+This repository contains only the frontend implementation of the Author's Haven project. You can click **[here](https://github.com/andela/selene-ah-backend)** to check the backend implementation.
+
+Start by cloning the repository to your local machine
+
+```bash
+git clone https://github.com/andela/selene-ah-frontend.git
+
+cd selene-ah-frontend
+```
+
+Next install the package dependencies by running
+
+```bash
+npm install
+```
+
+After that start or build the application by running
+
+```bash
+  npm start # For a local development server
+  npm serve # For production
```

Comment:
This should be `npm run serve` @ekpangmichael. If not it will throw an error.

Reply:
@ekpangmichael has this been addressed?

Reply:
Working on it

Reply:
I also think this is redundant because `npm start` is running `npm run serve` according to your `package.json` file. You could remove the `npm run serve` you added in the `README.md` file.